src/kernels/execution/kernelExecution.ts

```diff
@@ -1,342 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
```

Comment:
This is now not valid
Interrupt and restart are part of the kernel
& we have a dedicated execution class for `executeCode`